source/platforms/BitBucketServer.ts

```diff
@@ -39,18 +39,32 @@ export class BitBucketServer implements Platform {
    * Fails the current build, if status setting succeeds
    * then return true.
    */
-  updateStatus = async (passed: boolean | "pending", message: string, url?: string): Promise<boolean> => {
+  updateStatus = async (
+    passed: boolean | "pending",
+    message: string,
+    url?: string,
+    dangerID?: string
+  ): Promise<boolean> => {
     const pr = await this.api.getPullRequestInfo()
     const { latestCommit } = pr.fromRef
+
     let state = passed ? "SUCCESSFUL" : "FAILED"
     if (passed === "pending") {
       state = "INPROGRESS"
     }
+
+    let name = "Danger"
+    if (process.env["PERIL_INTEGRATION_ID"]) {
+      name = "Peril"
```

Comment:
Do we need to support this use case in Peril? I'm not familiar enough with it to know if it's possible to run multiple instances.

Reply:
Nope,  or at least whatever we need to do it'd be a bit of work on Perils side 👍